pymemcache/client/base.py

```diff
@@ -705,8 +705,12 @@ def _raise_errors(self, line, name):
             raise MemcacheServerError(error)
 
     def _fetch_cmd(self, name, keys, expect_cas):
-        checked_keys = dict((self.check_key(k), k) for k in keys)
-        cmd = name + b' ' + b' '.join(checked_keys) + b'\r\n'
+        if name == b'stats':
```

Comment:
Can we add a comment here explaining why the stats command warrants space splitting, for future readers.